elasticsearch-core/src/test/scala/com/sumologic/elasticsearch/restlastic/RestlasticSearchClientTest.scala

```diff
@@ -51,7 +51,7 @@ class RestlasticSearchClientTest extends WordSpec with Matchers with ScalaFuture
       val analyzer = Analyzer(analyzerName, Keyword, Lowercase)
       val indexSetting = IndexSetting(12, 1, analyzer)
       val indexFut = restClient.createIndex(index, Some(indexSetting))
-      whenReady(indexFut) { _ => refresh() }
```

Comment:
yeah this is fine -- you could have used refreshWithClient everywhere, but I don't really care.